src/index.html

```diff
@@ -160,7 +160,7 @@ <h3>New furniture</h3>
         </div>
       </div>
       <div class="row">
-        <div class="col-3">
+        <div class="col-xl-3 col-lg-4 col-md-6 col-sm-12">
```

Comment:
Looks good to me, but please keep in mind that you don't cover the interval less than `sm`; it means from 0 to 576px. According to the docs it can be achieved by using `col-N` classes: https://getbootstrap.com/docs/4.2/layout/grid/#grid-options